src/test/java/htsjdk/samtools/CRAMFileReaderTest.java

```diff
@@ -42,13 +43,17 @@
 public class CRAMFileReaderTest {
 
     private static final File TEST_DATA_DIR = new File("src/test/resources/htsjdk/samtools");
+    private static final File CRAM_FILE = new File(TEST_DATA_DIR, "cram_with_crai_index.cram");
```

Comment:
Can you rename this to something like CRAM_WITH_CRAI, to make the tests read a little easier.